addons/mass_mailing/models/mailing_contact.py

```diff
@@ -20,6 +20,7 @@ class MassMailingContactListRel(models.Model):
     list_id = fields.Many2one('mailing.list', string='Mailing List', ondelete='cascade', required=True)
     opt_out = fields.Boolean(string='Opt Out',
                              help='The contact has chosen not to receive mails anymore from this list', default=False)
+    subscription_date = fields.Datetime(string='Subscription Date', readonly=True)
```

Comment:
Why not using create_date ? Seems to use the same meaning without adding new fields.